README.md

```diff
@@ -147,6 +147,7 @@ For the moment, we are currently focused on the CISA KEV Database and Google Tsu
 
 | CVE ID                                                  | Implemented | Detail                                                  | Published Date |
 |:--------------------------------------------------------|:-----------:|:--------------------------------------------------------|:--------------:|
+| CVE-2024-43399                                          |      ✅      | Custom Exploit by Ostorlab: included in Agent Asteroid. |   2024-08-19   |
```

Comment:
Shouldn't be merged until the exploit is merged

Reply:
why?